scout/server/extensions/loqus_extension.py

```diff
@@ -279,18 +278,13 @@ def get_exec_loqus_variant(self, loqus_instance, variant_info):
                     variant_info["variant_type"],
                 ]
             )
-        output = ""
         try:
             output = execute_command(cmd)
-        except CalledProcessError as err:
-            LOG.warning("Something went wrong with loqus")
-            raise err
-
-        res = {}
-        if output:
-            res = json.loads(output)
-
-        return res
+            if output is not None:
+                return json.loads(output)
+        except CalledProcessError:
```

Comment:
I've simulated the error by actually raising the CalledProcessError in a connected local instance of loqusdb (exe):

![image](https://user-images.githubusercontent.com/28093618/129341405-5e8dea76-6f92-4a8a-b1fc-16b1dbf4cc39.png)


The error is propagated to Scout but the app doesn't crash: 👍🏻 

![image](https://user-images.githubusercontent.com/28093618/129341311-c087f249-ec4b-4413-a837-f78ac51eb30f.png)